.github/workflows/python_sdk_publish.yaml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  push:
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+
+      - name: Install Poetry
+        run: |
+          curl -sSL https://install.python-poetry.org | python - -y
```

Comment:
Let's use `actions/python-setup/action.yaml` instead, it sets up the correct Python version as configured in the repo, installs Poetry, etc.